.rubocop_todo.yml

```diff
@@ -1,104 +1,165 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-11-22 09:25:28 +0100 using RuboCop version 0.51.0.
+# on 2018-02-09 10:14:18 +0000 using RuboCop version 0.51.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 2
-# Configuration parameters: AllowSafeAssignment.
-Lint/AssignmentInCondition:
+# Cop supports --auto-correct.
+# Configuration parameters: SupportedStyles, IndentationWidth.
+# SupportedStyles: special_inside_parentheses, consistent, align_brackets
+Layout/IndentArray:
+  EnforcedStyle: consistent
+
+# Offense count: 4
+# Cop supports --auto-correct.
+Layout/SpaceAfterComma:
   Exclude:
-    - 'app/models/place_attributes.rb'
+    - 'spec/factories/api/v3/api_v3_map_attributes.rb'
 
-# Offense count: 35
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, SupportedStylesForEmptyBraces.
+# SupportedStyles: space, no_space
+# SupportedStylesForEmptyBraces: space, no_space
+Layout/SpaceBeforeBlockBraces:
+  Exclude:
+    - 'spec/services/api/v3/flows/filter_spec.rb'
+
+# Offense count: 2
+Lint/AmbiguousOperator:
+  Exclude:
+    - 'spec/factories/api/v3/api_v3_country_properties.rb'
+
+# Offense count: 2
+Lint/RescueWithoutErrorClass:
+  Exclude:
+    - 'app/workers/database_update_worker.rb'
+    - 'lib/tasks/db_remote_import.rake'
+
+# Offense count: 44
 Metrics/AbcSize:
-  Max: 67
+  Max: 51
 
-# Offense count: 10
+# Offense count: 18
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Max: 126
+  Max: 83
 
-# Offense count: 6
+# Offense count: 10
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 611
+  Max: 279
 
 # Offense count: 8
 Metrics/CyclomaticComplexity:
-  Max: 19
+  Max: 11
 
-# Offense count: 465
+# Offense count: 211
 # Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
 # URISchemes: http, https
 Metrics/LineLength:
-  Max: 1095
-
-# Offense count: 44
+  Max: 80
```

Comment:
this is a bit more changes than I would expect. Are we addressing past issues too?